global.R

```diff
@@ -9,6 +9,7 @@
 # renv::status()
 # Library calls ---------------------------------------------------------------------------------
 shhh <- suppressPackageStartupMessages # It's a library, so shhh!
+shhh(library(git2r, pos = 10000))
```

Comment:
Can remove git2r now, as it's not needed after this morning's update to the commit hooks.